config.js

```diff
@@ -28,12 +28,12 @@ var config = module.exports.config = {};
  * @param  {String}    path            The path to the UI static assets
  */
 config.ui = {
-    'path': '../3akai-ux'
+    'path': './3akai-ux'
```

Comment:
@davidoae can you have a look at this? We need to make sure it doesn't mess things up.